update: email aliasing SimpleLogin section to reflect latest Proton integration status (#3058)
privacyguides/privacyguides.org: docs/email-aliasing.md

</div>

> SimpleLogin was [acquired by Proton AG](https://proton.me/news/proton-and-simplelogin-join-forces) as of April 8, 2022. If you use [Proton Mail](https://proton.me/mail/pricing) and [Proton Pass](https://proton.me/pass/aliases) (excluding the Proton Mail Free plan) for your primary mailbox, SimpleLogin's features have been incorporated as "[hide-my-email aliases](https://proton.me/blog/hide-my-email-aliases)" and is a great choice. As both products are now owned by the same company you now only have to trust a single entity. We also expect that SimpleLogin will be more tightly integrated with Proton's offerings in the future. SimpleLogin continues to support forwarding to any email provider of your choosing.

Reviewer: ```suggestion
SimpleLogin was [acquired by Proton AG](https://proton.me/news/proton-and-simplelogin-join-forces) as of April 8, 2022. Thus, SimpleLogin is a great choice if you use [Proton Mail](email.md#proton-mail) for your primary mailbox, because you now only have to trust a single entity. However, SimpleLogin continues to support forwarding to any email provider of your choosing.
```
This is more of a Proton Pass feature than a SimpleLogin feature, and the integration with Proton Mail is still fairly lacking, because you still can't easily send _from_ aliases in Proton Mail :(

I'd rather shorten this paragraph up and maybe mention this feature under Proton Mail's (or Pass's) listing in the future, but only if the integration gets better than it currently is. When Proton Mail lets you generate reply-from aliases in the message composer that will be more relevant.

Reviewer: Right but you can't send a new email from an alias easily because there is no composer integration.

Author: Ah, I see. Sure... but I actually don't think I have a need for that?

The point of email aliasing, to me, is so that all the random websites that demand an email address for no reason other than they can to not be getting any genuine email addresses that can be linked behind the scenes.

But if I need to be regularly sending from an email alias then this is unlikely to be shopping or social media or anything like that... and... well... in all of those situations the person on the other end already knows me and so I create a real email address (e.g. my.useful.address@provider.com) and use a proper email alias (of which I can have 15 on my Proton Mail plan) rather than one of these login aliases.

What's the use case for needing a "spoof" alias where you also need to be able to initiate email chains?

Reviewer 2: ```suggestion
SimpleLogin was [acquired by Proton AG](https://proton.me/news/proton-and-simplelogin-join-forces) on April 8, 2022, making it an excellent option for users of [Proton Mail](email.md#proton-mail) as their primary mailbox. With this acquisition, you only need to trust a single entity for your email services. SimpleLogin still allows forwarding to any email provider of your choice.
```

While not grammatically incorrect, I try to avoid starting sentences with conjunctive adverbs (thus, however, etc) because it's kind of unnecessary when you have a full stop.

I think this sounds a little better.

Reviewer 3: i like to add that we should not write expected features but only actual state.

> SimpleLogin was [acquired by Proton AG](https://proton.me/news/proton-and-simplelogin-join-forces) as of April 8, 2022. If you use [Proton Mail](https://proton.me/mail/pricing)) and [Proton Pass](https://proton.me/pass/aliases) (excluding the Proton Mail Free plan) for your primary mailbox, SimpleLogin's features have been incorporated as "[hide-my-email aliases](https://proton.me/blog/hide-my-email-aliases)" and is a great choice. As both products are now owned by the same company you now only have to trust a single entity. We also expect that SimpleLogin will be more tightly integrated with Proton's offerings in the future. SimpleLogin continues to support forwarding to any email provider of your choosing.

Author: Fixed in #7624ca2

> You can link your SimpleLogin account in the settings with your Proton account. If you have Proton Pass Plus, Proton Unlimited, or any multi-user Proton plan, you will have SimpleLogin Premium for free and [vice versa](https://simplelogin.io/blog/sl-premium-including-pass-plus/). You can also purchase a voucher code for SimpleLogin Premium anonymously via their official reseller [ProxyStore](https://simplelogin.io/faq).

Reviewer: ```suggestion
You can link your SimpleLogin account in the settings to your Proton account. If you have Proton Pass Plus, Proton Unlimited, or any multi-user Proton plan, you will receive SimpleLogin Premium for free. You can also purchase a voucher code for SimpleLogin Premium anonymously via their official reseller [ProxyStore](https://simplelogin.io/faq). If you have SimpleLogin Premium without a Proton plan that includes Proton Pass, [you will receive Proton Pass Plus](https://simplelogin.io/blog/sl-premium-including-pass-plus/) for free.
```
"And vice versa" is confusing because we talk about 3 separate Proton plans in the sentence prior.

Author: Hopefully improved this with my latest commit
